uiowa/uiowa
# Re-enable ability to remove collections items

Resolves #3462, relates to #3401.





# How to test

`git checkout remove-collections-fix`
`git pull`
`blt frontend`
Sync a site with existing collections, or create new collections
Try to remove a collection item using the dropdown toggle
Successfully remove a collection item!

Also test sliders, and ensure that you are able to add/remove slides
